src/commands/autocomplete.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import Command, {flags} from 'cli-engine-command'
+import path from 'path'
+import {CustomColors} from 'cli-engine-command/lib/output'
+
+export default class Autocomplete extends Command {
+  static topic = 'autocomplete'
+  static description = 'autocomplete installation instructions'
+  static hidden = true
+  static flags = {
+    commands: flags.boolean({hidden: true}),
+    script: flags.boolean({hidden: true}),
+    shell: flags.string({description: 'shell to use', char: 's'})
+  }
+
+  async run () {
```

Comment:
can you write a test to at least execute this code? Seeing a lot of red from codecov right now